packages/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -1122,31 +1120,6 @@ describe('validation logic', () => {
                 ]
           );
         }
-
-        // Implicit casting of literal values tests
-        testErrorsAndWarnings(`from a_index | where numberField ${op} stringField`, [
-          `Argument of [${op}] must be [number], found value [stringField] type [string]`,
-        ]);
-        testErrorsAndWarnings(`from a_index | where stringField ${op} numberField`, [
-          `Argument of [${op}] must be [number], found value [stringField] type [string]`,
-        ]);
-        testErrorsAndWarnings(`from a_index | where numberField ${op} "2022"`, []);
-
-        testErrorsAndWarnings(`from a_index | where dateField ${op} stringField`, [
-          `Argument of [${op}] must be [string], found value [dateField] type [date]`,
-        ]);
-        testErrorsAndWarnings(`from a_index | where stringField ${op} dateField`, [
-          `Argument of [${op}] must be [string], found value [dateField] type [date]`,
-        ]);
-        testErrorsAndWarnings(`from a_index | where dateField ${op} "2022"`, []);
-
-        // Check that the implicit cast doesn't apply for fields
-        testErrorsAndWarnings(`from a_index | where stringField ${op} 0`, [
-          `Argument of [${op}] must be [number], found value [stringField] type [string]`,
-        ]);
-        testErrorsAndWarnings(`from a_index | where stringField ${op} now()`, [
-          `Argument of [${op}] must be [string], found value [now()] type [date]`,
-        ]);
```

Comment:
These felt superfluous, especially because I think the tests will soon be reorganized around functions more than commands (see https://github.com/elastic/kibana/issues/182390) and this is just a duplication of some tests that already exist in the `from` block